### ISSUE TITLE
Performance issues

### ISSUE THREAD

**Issue by gen2brain** [JDaren/subtitleConverter#20]
On android, parsing of just one srt file took longer then 30 seconds. After I changed all += string operations to use StringBuilder it was down to couple of seconds.

**Comment by JDaren** [comment]
Woaw, that much difference? Nowadays that would have been my initial solution, but at the time, I wasn't very focused on performance. Just happy it would work and in a PC the difference probably wasn't noticeable.

Feel free to submit a pull request, I don't know when I'll have to revisit this project and refactor the code myself to bring up to date.

**Comment by gen2brain** [comment]
Yes, that much. I am not expert for Java so I can't explain, just playing with android. But this change made it fly, and before that it was crawling. Huge difference. I will send pull request.

**Comment by JDaren** [comment]
String concatenation creates new objects and copies everything for each concatenation, on such text files as the subtitles, it is very bad practice to use concatenation since it will need to rewrite the whole content of the string in a new larger string each time a concatenation occurs. Simple consecutive concatenation is taken care of by the compiler,  but not for such big logic. Of course, I wasn't so ware of this at the time.
